conveyor/src/main/java/com/enfint/conveyor/service/CalculateFullRatingService.java

```diff
@@ -0,0 +1,167 @@
+package com.enfint.conveyor.service;
+
+import com.enfint.conveyor.dto.EmploymentDTO;
+import com.enfint.conveyor.dto.ScoringDataDTO;
+import com.enfint.conveyor.enumModel.EmploymentStatus;
+import com.enfint.conveyor.enumModel.MaritalStatus;
+import com.enfint.conveyor.enumModel.Position;
+import com.enfint.conveyor.exception.RefusalException;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.Period;
+
+import static com.enfint.conveyor.enumModel.EmploymentStatus.*;
+import static com.enfint.conveyor.enumModel.EmploymentStatus.BUSINESS_OWNER;
+import static com.enfint.conveyor.enumModel.Gender.FEMALE;
+import static com.enfint.conveyor.enumModel.Gender.MALE;
+import static com.enfint.conveyor.enumModel.Position.MID_MANAGER;
+import static com.enfint.conveyor.enumModel.Position.TOP_MANAGER;
+import static java.math.BigDecimal.valueOf;
+
+@Service
+@NoArgsConstructor
```

Comment:
No-args constructor creates by default, so you don't need to use this annotation